data/presets/presets/amenity/icecream.json

```diff
@@ -0,0 +1,21 @@
+{
+    "icon": "ice-cream",
+    "fields": [
+        "address",
+        "building_area",
+        "opening_hours",
+        "takeaway",
+        "delivery"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "ice-cream"
+    ],
+    "tags": {
+        "amenity": "ice_cream"
+    },
+    "name": "Ice Cream"
```

Comment:
Let's call it "Ice Cream Shop".    Without shop, it just sounds like a pile of ice cream on the ground.